src/traces/scattercarpet/attributes.js

```diff
@@ -107,7 +107,8 @@ module.exports = {
         line: extendFlat({},
             {width: scatterMarkerLineAttrs.width},
             colorAttributes('marker'.line)
-        )
+        ),
+        gradient: scatterMarkerAttrs.gradient
```

Comment:
TODO: test image with carpet plots. Somehow `imagetest` isn't working for me yet with carpets, must be something I still need to update?